docs/decisions/0013-special-exam-submission-and-review-events.rst

```diff
@@ -0,0 +1,37 @@
+12. Event definitions for special exam post-submission and review
+#################################################################
+
+Status
+******
+
+**Draft** 2023-09-29
+
+Context
+*******
+
+* A new backend for exams called `edx-exams` is being developed (See the `exams IDA ADR <https://github.com/openedx/edx-proctoring/blob/master/docs/decisions/0004-exam-ida.rst>`_ for more info).
+* We are currently working to implement the downstream effects triggered whenever an exam attempt is submitted or reviewed. (For example, when an exam attempt is submitted, we will want to make sure `edx-platform` knows to mark the exam subsection as completed.)
+
+
+Decision
+********
+
+* Since `edx-exams` is independent, we will want to prevent circular dependencies between `edx-platform` and `edx-exams`.
```

Comment:
I think this isn't all that important from the perspective of someone working with events.